scripts/sct_utils.py

```diff
@@ -206,6 +206,99 @@ def run(cmd, verbose=1, error_exit='error', raise_exception=False):
+    """
+    Print the syntax to open a viewer and display images for QC. To use default values, enter empty string: ''
+    Parameters
+    ----------
+    l_file [list:string]: list of NIFTI file names
+    l_colormap [list:string]: list of colormaps associated with each file. Available colour maps: blue, blue-lightblue, cool, copper, cortical, green, greyscale, hot, hsv, pink, random, red, red-yellow, render1, render1t, render2, render2t, render3, retino, subcortical, yellow. Default=greyscale.
+    l_minmax [list:string]: list of min,max brightness scale associated with each file. Separate with comma.
+    l_opacity [list:string]: list of opacity associated with each file. Between 0 and 1.
+
+    Returns
+    -------
+    None
+
+    Example
+    -------
+    sct.display_viewer([file1, file2, file3])
+    sct.display_viewer([file1, file2], l_colormap=['gray', 'red'], l_minmax=['', '0,1'], l_opacity=['', '0.7'])
+    """
+    list_viewer = ['fslview', 'fslview_deprecated', 'fsleyes']  # list of known viewers. Can add more.
+    dict_fslview = {'gray': 'Greyscale', 'red-yellow': 'Red-Yellow', 'blue-lightblue': 'Blue-Lightblue', 'red': 'Red', 'random': 'Random-Rainbow'}
+    dict_fsleyes = {'gray': 'greyscale', 'red-yellow': 'red-yellow', 'blue-lightblue': 'blue-lightblue', 'red': 'red', 'random': 'random'}
+    selected_viewer = None
+
+    # find viewer
+    for viewer in list_viewer:
+        if check_exe(viewer):
+            selected_viewer = viewer
+            break
+    if selected_viewer == None:
+        # exit without error
+        return
+
+    # loop across files and build syntax
+    cmd = viewer
+    # add mode (only supported by fslview for the moment)
+    if mode and selected_viewer in ['fslview', 'fslview_deprecated']:
+        cmd += ' -m ' + mode
+    for i in range(len(l_file)):
+        # add viewer-specific options
+        if selected_viewer in ['fslview', 'fslview_deprecated']:
+            cmd += ' ' + l_file[i]
+            if l_colormap:
+                if l_colormap[i]:
+                    cmd += ' -l ' + dict_fslview[l_colormap[i]]
+            if l_minmax:
+                if l_minmax[i]:
+                    cmd += ' -b ' + l_minmax[i]  # a,b
+            if l_opacity:
+                if l_opacity[i]:
+                    cmd += ' -t ' + l_opacity[i]
+        if selected_viewer in ['fsleyes']:
+            cmd += ' ' + l_file[i]
+            if l_colormap:
+                if l_colormap[i]:
+                    cmd += ' -cm ' + dict_fsleyes[l_colormap[i]]
+            if l_minmax:
+                if l_minmax[i]:
+                    cmd += ' -dr ' + ' '.join(l_minmax[i].split(','))  # a b
+            if l_opacity:
+                if l_opacity[i]:
+                    cmd += ' -a ' + str(float(l_opacity[i])*100)  # in percentage
+    cmd += ' &'
+    # display
+    if verbose:
+        printv('\nDone! To view results, type:')
+        printv(cmd + '\n', verbose=1, type='info')
+    return None
```

Comment:
`return None` is useless

Reply:
fixed in a17d2bf1e1a506ef06b9de1732a22c867f5d2463